tutorials/object_gcr_4_photoz.ipynb

```diff
@@ -0,0 +1,264 @@
+    "\n",
+    "Owners: **Yao-Yuan Mao [@yymao](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@yymao)**  \n",
+    "Last Verifed to Run: **2019-02-22** (by @yymao)\n",
+    "\n",
+    "This notebook will show you how to access the \"add-on\" columns that provide the photometric redshift (photo-z) information for the DC2 Object Catalog (Run 1.2i). \n",
+    "\n",
+    "__Learning objectives__: After going through this notebook, you should be able to:\n",
+    "  1. Load and efficiently access a DC2 object catalog (+ photo-z) with the GCR\n",
+    "  2. Understand how the photo-z data are stored / represented\n",
+    "  3. Look at an example of galaxy photo-z distributions\n",
+    "  \n",
+    "__Logistics__: This notebook is intended to be run through the JupyterHub NERSC interface available here: https://jupyter-dev.nersc.gov. To setup your NERSC environment, please follow the instructions available here: https://confluence.slac.stanford.edu/display/LSSTDESC/Using+Jupyter-dev+at+NERSC\n",
+    "\n",
+    "__Other notes__: \n",
+    "If you restart your kernel, or if it automatically restarts for some reason, all imports and variables will become undefined so, you will have to re-run everything."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import GCRCatalogs\n",
+    "from GCR import GCRQuery"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the catalog\n",
+    "\n",
+    "Loading the object catalog with photo-z add-on. The catalog name is `dc2_object_run1.2i_with_photoz`. \n",
+    "\n",
+    "**Note**: if you need more quantities (including those not in DPDD), then use `dc2_object_run1.2i_all_columns_with_photoz` instead.\n",
+    "\n",
+    "It takes a few seconds for the catalog instance to initiate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cat = GCRCatalogs.load_catalog('dc2_object_run1.2i_with_photoz')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Photo-z access methods\n",
+    "\n",
+    "The photo-z infomation can be accessed in two ways:\n",
+    "\n",
+    "1. As a single column `photoz_mode` that has the mode value (z_peak), and\n",
+    "2. As a single multi-dimension column `photoz_pdf` \n",
+    "   (i.e., when accessing this column, you get a 2D array (objects x PDF values) instead of a 1D array). \n",
+    "   Note that the column contains only the PDF values, not the bin center values. \n",
+    "   The PDF bin centers are stored in a catalog attribute named `photoz_pdf_bin_centers`. \n",
+    "\n",
```

Comment:
Would it be useful to make a plot of the pdfs and then also have the mode value plotted as straight dashed lines (to show the consistency of the different approaches?)

Reply:
Implemented!